playbooks/tests/tasks/ceph.yml

```diff
@@ -13,3 +13,62 @@
   tasks:
   - name: cinder uses rbd_hybrid or rbd_ssd as backend
     shell: grep -P "^enabled_backends.+rbd_(hybrid|ssd)" /etc/cinder/cinder.conf
+
+- name: restart ceph osd service,and check ceph status
+  hosts: ceph_osds[0]
```

Comment:
I prefer to use `hosts: ceph_osds_ssd[0]:ceph_osds_hybrid[0]`
in case we will have CI with multi ceph backends.
We don't need serial: 1 here, ssd osd node and hybrid osd node can be restarted/rebooted at the same time.